experiment/measurer.py

```diff
@@ -588,7 +588,7 @@ def set_up_coverage_binary(benchmark):
     coverage_binaries_dir = build_utils.get_coverage_binaries_dir()
     benchmark_coverage_binary_dir = coverage_binaries_dir / benchmark
     if not os.path.exists(benchmark_coverage_binary_dir):
-        os.mkdir(benchmark_coverage_binary_dir)
+        os.makedirs(benchmark_coverage_binary_dir)
```

Comment:
consider using common.filesystem.create_dir()

Reply:
Good suggestion! Then the `non exist` check can also be deleted.
wdyt? @jonathanmetzman

Reply:
Done.